bot/pagination.py

```diff
@@ -87,6 +108,53 @@ def add_line(self, line: str = '', *, empty: bool = False) -> None:
             self._current_page.append('')
             self._count += 1
 
+        if remaining_words:
+            self._new_page()
+            self.add_line(remaining_words)
+
+    def _new_page(self) -> None:
```

Comment:
Docstrings, even for private helper methods? Yes please!

Reply:
I may have been a little imprecise here, but this is actually a request for a docstring :grin:

Reply:
Oh, I see. I'll add a brief docstring to `_new_page` :+1:.